modules/elb.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: module
+title: Elastic Load Balancing
+image: assets/img/ec2.png
+description: Create and manage Elastic Load Balancers
+---
+Overview
+--------
+Cumulus makes configuring Elastic Load Balancers simpler. Easily configure load balancers, listeners, and policies. Read the following sections to learn about configuring your load balancers with Cumulus. Example configuration can be found in the [Cumulus repo](https://github.com/lucidsoftware/cumulus).
+
+Load Balancer Definition
+-----------------------
+
+Each load balancer is defined in its own file where the file name is the name of the load balancer. These files are located in a [configurable](#configuration) folder. Load balancers are JSON objects with the following attributes:
+
+* `listeners` - the configuration for the listeners on the load balancer. Listeners can be defined using templates and/or inline (see [listeners](#listeners). At least one listener between `includes` and `inlines` must be present for a valid load balancer configuration. Has the following properties:
+  * `includes` - an optional array of definitions for the template listeners to include. Each definition has the following properties:
+    * `template` - the name of the template to include (without .json). The template should be located in the [configurable](#configuration) listeners directory.
+    * `vars` - an object whose key-value pairs will override variables in the template
+  * `inlines` - inline definitions for templates. See [listeners](#listeners) for detailed information
+* `subnets` - an array of strings describing what subnets to attach the load balancer to. These can either be the subnet id or the value of the "Name" tag on the subnet (Cumulus will always try to use the "Name" tag to figure out which subnet is being used and default to treating the value as a subnet id)
+* `security-groups` - an array of security group ids to apply to the load balancer
+* `internal` - a true/false value indicating if the load balancing is internal facing (has a scheme of "internal"). Cannot be updated after a load balancer is created
+* `tags` - an optional JSON object whose key/value pairs correspond to tag keys and values for the load balancer e.g. `{ "TagName": "TagValue" }`
+* `manage-instances` - if false or omitted, instances will not be registered or deregistered with the load blancer when you sync. It can also be an array of instance ids that will be synced where any instances missing from here will be deregistered from the load balancer, and any that are added will be registered to the load balancer
```

Comment:
It can also be an array of instance ids that will be synced such that any instances missing from the array will be deregistered from the load balancer, and any that are added will be registered to the load balancer